pkg/install/generate.go

```diff
@@ -375,6 +376,17 @@ func InstallerPodSpec(
 			},
 		)
 		env = append(env, oVirtCredsEnvVars(cd.Spec.Platform.Ovirt.CredentialsSecretRef.Name)...)
+	case cd.Spec.Platform.IBMCloud != nil:
+		env = append(env, corev1.EnvVar{
+			Name: constants.IBMCloudAPIKeyEnvVar,
+			ValueFrom: &corev1.EnvVarSource{
+				SecretKeyRef: &corev1.SecretKeySelector{
+					LocalObjectReference: cd.Spec.Platform.IBMCloud.CredentialsSecretRef,
+					Key:                  constants.IBMCloudAPIKeySecretKey,
+					Optional:             pointer.BoolPtr(true),
```

Comment:
Is it really optional to not set the env var for the installer?

Reply:
Not really optional. Updated this